code/game/jobs/job/cargo_service.dm

```diff
@@ -111,6 +111,7 @@ Bartender
 /datum/outfit/job/bartender
 	name = "Bartender"
 
+	glasses = /obj/item/clothing/glasses/sunglasses/reagent
```

Comment:
True, but he's already difficult to get to, as is, if he's hunkered down behind the bar--not to mention, is it really worth balancing a job role against Rev, especially when gang arguably a superior replacement?